apps/app/src/utils.ts

```diff
@@ -66,6 +66,41 @@ const unique = (array: any[], propertyName: string) => {
   );
 };
 
+/**
+ * This helper function will take in a provided input string and a substringToMatchOn
+ * and will return a list of parts of the input string and whether or not they match
+ * on any part of the substrcingToMatchOn
+ *
+ * This is useful for bolding parts of a search string that uses spaces.
+ */
+const getMatchingPartsFromSubstring = (
```

Comment:
the corresponding logic exists in elements which i feel like is a faux pas to add here, in the react app - so i've copied the logic and have it return a list of objects that can be mapped over to determine if the parts match any part of the search string or not.